tests/unit/challenges/test_views.py

```diff
@@ -1859,6 +1860,28 @@ def test_create_challenge_using_zip_file_when_user_is_not_authenticated(self):
         self.assertEqual(response.data.values()[0], expected['error'])
         self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
 
+    def test_create_challenge_using_zip_file_success(self):
+        self.url = reverse_lazy('challenges:create_challenge_using_zip_file',
+                                kwargs={'challenge_host_team_pk': self.challenge_host_team.pk})
+
+        assert Challenge.objects.count() == 1
```

Comment:
Instead of this, please use `self.assertEqual(Challenge.objects.count(), 1)`. Please do the same changes for other assert statements as well.